randovania/games/samus_returns/logic_database/Area 3 - Factory Interior.txt

```diff
@@ -315,31 +321,44 @@ Extra - asset_id: collision_camera_022
               All of the following:
                   High Jump Boots
                   Super Jump (Advanced) or Unmorph Extend (Beginner)
-  > Start Point
-      Trivial
 
-> Above Lava Pits; Heals? False
+> Next to Lava Pits; Heals? False
   * Layers: default
   > Door to Paraby Periphery
       All of the following:
           Varia Suit
           Any of the following:
-              Grapple Beam or Gravity Suit or Space Jump
+              Gravity Suit
               Power Bomb ≥ 2 and Spider Boost (Intermediate) and Can Spider Boost
-  > Door to Alpha Arena
+              All of the following:
+                  Use Spider Ball
+                  Grapple Beam or Space Jump
+  > Above Lava Pits
       All of the following:
-          Morph Ball and Varia Suit
-          Any of the following:
-              Grapple Beam or Space Jump or Simple IBJ
-              Spider Boost (Intermediate) and Can Spider Boost
-              Damage Boost (Intermediate) and Unmorph Extend (Intermediate)
+          Varia Suit
+          Wall Jump (Beginner) or Climb Rooms Vertically (High Jump)
 
 > Start Point; Heals? False; Default Node
   * Layers: default
   * Extra - start_point_actor_name: ST_SG_Alpha_001
   > Door to Alpha Arena
       Varia Suit
 
+> Above Lava Pits; Heals? False
+  * Layers: default
+  > Door to Alpha Arena
+      All of the following:
+          Morph Ball and Varia Suit
+          Any of the following:
+              Grapple Beam or Space Jump or Simple IBJ
+              Spider Boost (Intermediate) and Can Spider Boost
+              # https://youtu.be/M9CyPEmsFEA
+              Damage Boost (Advanced) and Unmorph Extend (Intermediate)
```

Comment:
is an unmorph extend actually needed here?

Reply:
Rewatching the video, its probably more accurate for this to be Mid Air Morph instead

Reply:
yeah I figured it might've been intended to be that. nothing else stuck out to me.